phoenix-core/src/test/java/org/apache/phoenix/execute/MutationStateTest.java

```diff
@@ -175,4 +185,43 @@ public void testGetMutationBatchList() {
 
     }
 
+    @Test(expected = SQLException.class)
```

Comment:
Same comment as above. Let's assert the exact sqlexception code too.

Reply:
Same as above. Used ExcpetedExceptionRule